src/get_clades_to_model.py

```diff
@@ -57,21 +59,84 @@ def get_next_wednesday(starting_date: datetime) -> str:
     return next_wednesday.strftime("%Y-%m-%d")
 
 
-def main(round_id: str, clade_output_path: Path):
+def get_clades(
```

Comment:
The `get_clades` is [copied from cladetime](https://github.com/reichlab/cladetime/blob/main/src/cladetime/get_clade_list.py#L24). Once this PR is merged, we'll delete the cladetime version.